desktop/mac/permission-requirements.md

```diff
@@ -1,28 +1,29 @@
 ---
-description: Docker Desktop Privileged Helper on Mac
+description: Permission requirements for Docker Desktop for Mac
 keywords: Docker Desktop, mac, security, install
-title: Docker Desktop Privileged Helper on Mac
+title: Docker Desktop permission requirements for Mac
 redirect_from:
 - /docker-for-mac/privileged-helper/
+- /desktop/mac/privileged-helper/
 ---
 
 This page contains information about the permission requirements for running and installing Docker Desktop on Mac, the functionality of the privileged helper process `com.docker.vmnetd` and the reasoning behind this approach, as well as clarification about running containers as `root` as opposed to having `root` access on the host.
 
 ## Permission requirements
 
-While Docker Desktop does not generally require the user running or installing it to have `root` privileges, in the default use case it needs `root` access to be granted on the first run. The first time Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required. 
+In the default set up flow, Docker Desktop for Mac does not require root privileges for installation but does require `root` access to be granted on the first run. The first time that Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required.
```

Comment:
Minor suggestion: "The first time that Docker Desktop is launched the user gets an admin prompt to grant permissions to install a privileged helper service."